spec/requests/odk/form_rendering_spec.rb

```diff
@@ -93,6 +93,14 @@
           {left_qing_id: form.c[9].id, op: "neq", value: 55}
         ]
       )
+
+      form.c[10].constraints.create!(
+        accept_if: "all_met",
+        conditions_attributes: [
+          {left_qing_id: form.c[10].id, op: "eq", value: 10}
+        ],
+        rejection_msg_translations: {"en": "Custom rejection message."}
```

Comment:
This looks great. Think you can use a symbol key on the `rejection_msg_translations`, which would be more conventional. Try it?